depends/hosts/linux.mk

```diff
@@ -1,7 +1,7 @@
 linux_CFLAGS=-pipe
 linux_CXXFLAGS=$(linux_CFLAGS)
 
-linux_release_CFLAGS=-O2
+linux_release_CFLAGS=-O1
 linux_release_CXXFLAGS=$(linux_release_CFLAGS)
 
 linux_debug_CFLAGS=-O1
```

Comment:
I am happy that the release and debug CFLAGS are now the same. What are the other differences in debug; can we drop it entirely?